slices/openjdk-25-jre-headless.yaml

```diff
@@ -0,0 +1,181 @@
+package: openjdk-25-jre-headless
+
+essential:
+  - openjdk-25-jre-headless_copyright
+
+slices:
+  # A set of slices to run headless openjdk runtime
+  standard:
+    essential:
+      - openjdk-25-jre-headless_class-data-sharing
+      - openjdk-25-jre-headless_console
+      - openjdk-25-jre-headless_core
+      - openjdk-25-jre-headless_debug
+      - openjdk-25-jre-headless_jfr
+      - openjdk-25-jre-headless_management
+      - openjdk-25-jre-headless_prefs
+      - openjdk-25-jre-headless_rmi
+      - openjdk-25-jre-headless_security
+      - openjdk-25-jre-headless_tools
+
+  # List of classes required to create the Class Data Sharing archive
+  class-data-sharing:
+    essential:
+      - openjdk-25-jre-headless_core
+    contents:
+      /usr/lib/jvm/java-25-openjdk-*/lib/classlist:
+
+  # A minimal set of files to run a Java application
+  # excluded dependencies:
+  # - ca-certificates-java_essential - needs chisel support to run
+  #                                    maintainer scripts.
+  # - java-common                    - provides update-alternatives,
+  #                                    not relevant.
+  # - util-linux                     - needed for bash completion
+  #                                    not relevant.
+  # - libjpeg8                       - used in awt, not relevant
+  # - liblcms2-2                     - used in awt, not relevant
+  core:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libstdc++6_libs
+      - zlib1g_libs
+    contents:
+      /etc/java-25-openjdk/jaxp.properties:
+      /etc/java-25-openjdk/jvm-*.cfg:
+      /etc/java-25-openjdk/logging.properties:
+      /etc/java-25-openjdk/net.properties:
+      /etc/java-25-openjdk/security/java.policy:
+      /etc/java-25-openjdk/security/java.security:
+      /usr/lib/jvm/java-25-openjdk-*/bin/java:
+      /usr/lib/jvm/java-25-openjdk-*/conf/jaxp.properties:
+      /usr/lib/jvm/java-25-openjdk-*/conf/logging.properties:
+      /usr/lib/jvm/java-25-openjdk-*/conf/net.properties:
+      /usr/lib/jvm/java-25-openjdk-*/conf/security/java.policy:
+      /usr/lib/jvm/java-25-openjdk-*/conf/security/java.security:
+      /usr/lib/jvm/java-25-openjdk-*/lib/jexec:
+      /usr/lib/jvm/java-25-openjdk-*/lib/jspawnhelper:
+      /usr/lib/jvm/java-25-openjdk-*/lib/jvm.cfg:
+      /usr/lib/jvm/java-25-openjdk-*/lib/libextnet.so:
```

Comment:
there is also `lib/libfallbackLinker.so` in `armhf` and `i386` but unsure whether you want it in this slice